### DISCUSSION TITLE
How to integrate payment and react such as paypal

### DISCUSSION THREAD

**Discussion by PassionateUnity** [community/community#60085] [Programming Help]
### Select Topic Area

Question

### Body

Hello Everybody I want to your help what integration with react typescript and payment method such as payoneer and paypal. Is there anybody know that? Hope to discuss this problem. As you can see, This problem is always happened.

**Comment by tedster0629** [comment]
Hi. I have found that this url. You can use this url https://codepen.io/skoliver89/full/xxKQeOq. Hope to work as good. If you have problem, you should be contact me. Thank you.

**Reply by PassionateUnity** [reply]
Thank you for your reply. But there is problem is that react component...

**Reply by tedster0629** [reply]
Yeah. So you should change it.

**Reply by lfinzel** [reply]
oops tedstar!!!

**Comment by PassionateUnity** [comment]
Thank you tedster

**Comment by lfinzel** [comment]
oops tedstar

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @PassionateUnity , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by kartikxisk** [comment]
You can check out the official documentation.
https://developer.paypal.com/docs/checkout/standard/customize/single-page-app/

**Comment by tedster0629** [comment]
yeah. thank you but I don't know how to unsubscribe. please teach me. thank
you again
